ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -86,25 +90,24 @@ cat << EOF > ~/ssh-connection-workaround.yml
   hosts: all
   tasks:
   - name: Try to grab file to see install finished
-    shell: kcli scp root@cnfdc5-installer:/root/cluster_ready.txt /home/tester/vm_ready.txt
+    shell: >-
+      kcli scp root@cnfdc5-installer:/root/cluster_ready.txt /home/kni/vm_ready.txt &&
+      ls /home/kni/vm_ready.txt
+    register: result
+    until: result is success
+    retries: 180
+    delay: 60
   - name: Check if successful
-    stat: path=/home/tester/vm_ready.txt
+    stat: path=/home/kni/vm_ready.txt
```

Comment:
as this is now on the HV, maybe have a better name for it
like us_cluster_<name>.is_ready

Reply:
ack